src/Yandex/Allure/PHPUnit/Adapter/AllureAdapter.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Yandex\Allure\Adapter;
+namespace Yandex\Allure\PHPUnit\Adapter;
```

Comment:
```suggestion
namespace Yandex\Allure\PHPUnit;
```

Reply:
@baev does it look better now? (can you see suggested code?)

Reply:
I like `PhpUnit ` more than `PHPUnit ` :)

Reply:
👍 

PS Any thoughts on replacing `PHPUnit` with `PhpUnit`?

Reply:
@okolesnyk will update the code shortly 
@baev thank you for review